bindinfo/handle.go

```diff
@@ -636,6 +636,21 @@ func (cf *captureFilter) isEmpty() bool {
 	return len(cf.tables) == 0 && len(cf.users) == 0
 }
 
+// ParseCaptureTableFilter checks whether this filter is valid and parses it.
+func ParseCaptureTableFilter(tableFilter string) (f tablefilter.Filter, valid bool) {
+	// forbidden wildcards '!' and '@' for safety,
+	// please see https://github.com/pingcap/tidb-tools/tree/master/pkg/table-filter for more details.
+	if strings.ContainsAny(tableFilter, "!@") {
```

Comment:
It's possible that a table name contains `!` or `@`.
According to the implementation of `(*tableRulesParser).parse()`, I think we can first `strings.TrimLeft(tableFilter, " \t")`, then check if the first character is `!` or `@`.

Reply:
it has been fixed, PTAL